tests/tests/test_IO/test_cell_io.py

```diff
@@ -1,303 +1,343 @@
-    332,
-    134,
-    133,
-    134,
-    132,
-    133,
-    136,
-    136,
-    130,
-    1026,
-    336,
-    134,
-    134,
-    336,
-    343,
-    3427,
-    2012,
-    138,
-    138,
-    140,
-    135,
-    134,
-    137,
-    142,
-    139,
-    1045,
-    141,
-    2045,
-    1048,
-    142,
-    142,
-    145,
-    150,
-    151,
-    151,
-    151,
-    146,
-    147,
-    147,
-    147,
-    1068,
-    149,
-    521,
-    522,
-    523,
-    523,
-    525,
-    520,
-    520,
-    521,
-    522,
-    524,
-    522,
-    526,
-    525,
-    525,
-    526,
-    524,
-    974,
-    522,
-    527,
-    524,
-    523,
-]
-z_vals = [
-    1282,
-    1276,
-    1281,
-    1286,
-    1276,
-    1278,
-    1277,
-    1278,
-    1285,
-    1297,
-    1274,
-    1294,
-    1290,
-    1279,
-    1280,
-    1296,
-    1290,
-    1294,
-    1294,
-    1285,
-    1279,
-    1289,
-    1275,
-    1286,
-    1281,
-    1284,
-    1275,
-    1291,
-    1273,
-    1294,
-    1294,
-    1274,
-    1274,
-    1274,
-    1283,
-    1287,
-    1290,
-    1294,
-    1272,
-    1275,
-    1275,
-    1274,
-    1292,
-    1273,
-    1288,
-    1286,
-    1287,
-    1289,
-    1298,
-    1274,
-    1274,
-    1273,
-    1275,
-    1275,
-    1278,
-    1286,
-    1278,
-    1288,
-    1295,
-    1279,
-    1282,
-    1275,
-    1276,
-    1275,
-    1286,
-]
-
-cubes_cells = [
-    Cell([340, 1004, 15], 1),
-    Cell([340, 1004, 15], 1),
-    Cell([392, 522, 10], 1),
-    Cell([392, 522, 10], 1),
-]
-
-roi_sorter_cells = [
-    Cell([4056, 564, 358], 1),
-    Cell([3989, 267, 570], 1),
-    Cell([4351, 735, 439], 1),
-    Cell([4395, 677, 367], 1),
-]
-
-
-def test_get_cells():
+
+@pytest.fixture
+def xml_path(data_path):
+    return str(data_path / "cells" / "cells.xml")
+
+
+@pytest.fixture
+def yml_path(data_path):
+    return str(data_path / "cells" / "cells.yml")
+
+
+@pytest.fixture
+def cubes_dir(data_path):
+    return str(data_path / "cube_extract" / "cubes")
+
+
+@pytest.fixture
+def roi_sorter_output_dir(data_path):
+    return str(data_path / "IO" / "roi_sorter_output")
+
+
+@pytest.fixture
+def type_vals():
+    return [
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+    ]
+
+
+@pytest.fixture
+def x_vals():
+    return [
```

Comment:
I agree! I just tried reformatting this though, and unfortunately `black` reformats it to one-per-line during `pre-commit`.

Reply:
Seems like it's a setting that can't be adjusted either https://github.com/psf/black/issues/1281

Reply:
Wonderful :sweat_smile: Do we already have a `data_dir` for this suite of tests? Because we could just `np.loadtxt` instead from some nicer-formatted `.csv` files instead to work around this, I guess?

Reply:
True - that's probably a better solution. I could also wrap these with `#fmt: on/off` to [exclude them from formatting](https://black.readthedocs.io/en/stable/usage_and_configuration/the_basics.html#ignoring-sections) which might be the easiest way to avoid this.

Reply:
`#fmt:on/off` would be a quick fix, but I think I'm leaning towards the `csv` solution just because:
- It'll only be a matter of time before someone makes an edit and moves/deletes these tags
- The values returned by the fixtures aren't intuitive -they're unlike the other fixtures, which return objects that are easily constructed and clearly take values to serve an intuitive purpose. IE More like some data that we load and choose to "believe".
- I'm guessing changes to these values would make our tests fail, which is very easy to do in such a large array and in a `.py` file that will have tests added/removed in the future. So I feel like they should be loaded from some data file that we agree to never touch.